lib/mongo_client.js

```diff
@@ -391,6 +391,7 @@ MongoClient.connect = function(url, options, callback) {
  * Starts a new session on the server
  *
  * @param {SessionOptions} [options] optional settings for a driver session
+ * @param {boolean} [options.causalConsistency=true] Enables or disables causal consistency for the session.
```

Comment:
`causalConsistency` is already defined on `SessionOptions`. We do not need to duplicate it here.